pkgs/tools/security/grap/default.nix

```
@@ -0,0 +1,48 @@
+{ lib, stdenv, fetchFromGitHub, boost, libseccomp, flex, python3Packages, swig4, bison, cmake, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "grap";
+  version = "1.3.1";
+
+  src = fetchFromGitHub {
+    owner = "QuoSecGmbH";
+    repo = "grap";
+    rev = "v${version}";
+    sha256 = "1fkdi7adfffxg1k4h6r9i69i3wi93s44c1j4cvr69blxsfh0mcnc";
+  };
+
+  nativeBuildInputs = [
+    boost.all
+    libseccomp
+    flex
+    python3Packages.pefile
+    python3Packages.pyelftools
+    python3Packages.capstone
+    swig4
+    bison
+    cmake
+  ];
```

```suggestion
  nativeBuildInputs = [
    bison
    cmake
    flex
    python3
  ];
  
  buildInputs = [
    boost.all
    libseccomp
    swig4
  ];
  
  strictDeps = true;
```

otherwise cross compile is broken. ``strictDeps = true;`` breaks wrapping which means the package is not done correct. I think you need to convert the package to use `buildPythonPackage` or use `wrapPythonPrograms`. I think the first option should be used.

---

I don't know if `buildPythonPackage` is used for things that also have a lot of C++, but the suggested change above about `strictDeps` and splitting the inputs looks great to me.